exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/StorageResources.java

```diff
@@ -169,6 +170,17 @@ public JsonResult enablePlugin(@PathParam("name") String name, @PathParam("val")
     }
   }
 
+  /**
+   * @deprecated use the method with POST request {@link #enablePlugin} instead
```

Comment:
I recommend remove all the methods of marked `@Deprecated` in this class. Help with code cleanup.

Reply:
Possibly it is better to do it after one major release. Because of this is REST API, some tools can still use old api